<launch>
      <!-- load the robot_description parameter before launching ROS-I nodes -->
    <include file="$(find workcell_moveit_config)/launch/planning_context.launch" >
        <arg name="load_robot_description" value="true" />
    </include>
    
    <node name="Fake_initial_position" type="Fake_initial_position.py" pkg="pa_ctrl" output="screen"/>
    <node name="Fake_Force_client" type="Fake_Force_client.py" pkg="pa_ctrl" output="screen"/>
    <node name="admittance" type="admittance.py" pkg="pa_ctrl" output="screen"/>
    <node name="descartes_node" type="descartes_node" pkg="myworkcell_core" output="screen"/>
</launch>